dev-haskell/random-build/metadata.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="project">
+		<email>haskell@gentoo.org</email>
+		<name>Gentoo Haskell</name>
+	</maintainer>
+	<use>
+		<flag name="pedantic">Enable -Werror</flag>
+	</use>
+	<upstream>
+		<remote-id type="hackage">random-build</remote-id>
+		<remote-id type="github">gentoo-haskell/random-build</remote-id>
+	</upstream>
```

Comment:
Looks like the `pedantic` flag was left in `metadata.xml`. Also, it's adding `remote-id type="hackage"` (there is a `--not-on-hackage` flag for `hackport make-ebuild` that keeps it from generating this).

```suggestion
	<upstream>
		<remote-id type="github">gentoo-haskell/random-build</remote-id>
	</upstream>
```